Test/Utils/ContrailAPI_New/VirtualNetwork.ps1

```diff
@@ -0,0 +1,9 @@
+# Dotsource needed files above this line
```

Comment:
Have you considered having just one 'header' file, instead of 1 per type? What conclusions did you reach and why?